python/mxnet/ndarray/numpy/_op.py

```diff
@@ -39,7 +39,7 @@
            'around', 'hypot', 'rad2deg', 'deg2rad', 'unique', 'lcm', 'tril', 'identity', 'take',
            'ldexp', 'vdot', 'inner', 'outer', 'equal', 'not_equal', 'greater', 'less', 'greater_equal', 'less_equal',
            'hsplit', 'rot90', 'einsum', 'true_divide', 'nonzero', 'shares_memory', 'may_share_memory', 'diff', 'resize',
-           'nan_to_num']
+           'nan_to_num', 'polyval']
```

Comment:
Do not add to the end of the list, could lead to unnecessary conflicts with other people's PRs. Suggest moving to somewhere like after `take`.

Reply:
Also remember to move the function definition.